ruby_on_rails/configure_percy.md

```diff
@@ -0,0 +1,33 @@
+# Configure Percy
+
+It's now time to configure Percy.
+
+__Create a new project on the percy website.__
+
+1. Visit [this](https://percy.io/organizations/renuo/projects/new) link.
+1. Give the project a name and select the github repository of the project.
+If you can't find the github repository in the dropdown you need to ask wg-operations to add the project to percy first.
+
+__Setup the CI__
+
+1. Add the PERCY_TOKEN and PERCY_PROJECT env variables to your CI.
+
+__Setup your Rails application__
+
+1. Add the gem `percy-capybara` to your gemfile and run `bundle install`.
+1. Add the following 2 lines to the `RSpec.configure do |config|` block in your `spec_helper.rb` file.
```

Comment:
this part is already explained in percy documentation. don't repeat it here but point to the docs